.gitignore

```diff
@@ -1,3 +1,5 @@
 .vscode/
 .DS_Store
 build/
+venv/
+venv_specification/
```

Comment:
Not sure I would add those. They are very specific to one's set up. Personally, I have all of my virtual environments under `~/virtualenv`, so not within a repo as such.

Reply:
We literally tell people to use `venv_specification` in the README, 

Less of a case for `venv`, except that it's pretty common